src/request.ts

```diff
@@ -152,6 +165,97 @@ export async function getRequestLinkDetails({
 	return x
 }
 
+export async function prepareXchainRequestLinkFulfillmentTransaction({
```

Comment:
thought: maybe lets name this `prepareXchainRequestFulfillmentTransaction({` ? How is it with x-chain claim?